src/lib/core/typography/_typography-utils.scss

```diff
@@ -34,6 +36,11 @@
   @return if($font-family == null, $font-family, unquote($font-family));
 }
 
+// Creates a CSS rule for font-family using the --mat-font-family css variable
+@mixin mat-font-family-css-var($config) {
+  @include mat-css-var(font-family, font-family, mat-font-family($config));
+}
```

Comment:
convenience method to wrap mat-font-family calls and provide the necessary css-variable css rules